editions/test/tiddlers/tests/data/transclude/Parameterised-Conditional-Parameters-Refreshed.tid

```diff
@@ -0,0 +1,39 @@
+title: Transclude/Parameterised/ConditionalParameters/Refreshed
+description: Parameterised transclusion when conditional parameters are changed by a refresh
+type: text/vnd.tiddlywiki-multiple
+tags: [[$:/tags/wiki-test-spec]]
+
+title: Output
+
+\procedure elephant(filler)
+\whitespace trim
```

Comment:
procedures now inherit `\whitespace trim` from the tiddler scope. So only 1 of them is needed

Reply:
I took that whitespace trim out, and the test started failing. The expect() error messages shows that a ton of whitespace is now present in the output.